### DISCUSSION TITLE
Add configuration file for every LXC/VM

### DISCUSSION THREAD

**Discussion by Gauvino** [BassT23/Proxmox#129] [Ideas]
My idea is to add a file for every LXC/VM extra update, so you can choose where do you want the script to look.
Ex: for octoprint path or pterodactyl or docker compose a thing like this and put the default path by default in the script

**Comment by pythongod** [comment]
I think via default it should only OS updates for VMs and LXCs. All APPs should be optional and it would be great to enable this per HOST
